offchain/danaswap-ui/views/components/footer.ejs

```diff
@@ -0,0 +1,34 @@
+<footer class="footer p-6">
+  <div class="columns is-vcentered is-mobile">
+    <div class="column">
+      <a href="/">
+        <img src="../../assets/images/logo_footer.png" alt="Ardana" />
+      </a>
+    </div>
+
+    <div class="column">
+      <div class="mt-6 is-pulled-right">
+        <div class="columns is-mobile">
+          <div class="column">
+            <img src="../../assets/images/twitter.png" />
```

Comment:
All images need `alt` tags, even if empty. That said, these aren't labeled at all. I would prefer to see `<img src="../../assets/images/twitter.png" alt="Twitter" title="Twitter">` with `title` because we have no reason to presuppose that users are familiar with these text-less logos (and would still prefer to see SVGs).